test/integration/consul-container/libs/node/consul-container.go

```diff
@@ -59,6 +63,7 @@ func NewConsulContainer(ctx context.Context, config Config) (Node, error) {
 		Mounts:       testcontainers.ContainerMounts{testcontainers.ContainerMount{Source: testcontainers.DockerBindMountSource{HostPath: configFile}, Target: "/consul/config/config.hcl"}},
 		Cmd:          config.Cmd,
 		SkipReaper:   skipReaper,
+		Env:          map[string]string{"CONSUL_LICENSE": license},
```

Comment:
Will this cause problems if set to blank?

Reply:
it shouldn't. in OSS we don't read it and in enterprise if blank it's an invalid license so it should fail as if the env var not declared.